packages/react/src/components/PageTitleBar/_page-title-bar.scss

```diff
@@ -198,6 +198,18 @@
       transition: opacity $duration--fast-01 linear, transform $duration--fast-01 linear;
       margin-left: 0;
 
+      // adding the current page as a breadcrumb makes it appear with the bx--link class, so we need
```

Comment:
Feels like "current page" is a standard breadcrumb scenario. Is it not possible to use the `isCurrentPage` prop of the BreadcrumbItem?

Reply:
I didn't know that was a prop. Thanks! It helps a little by making these overrides for link unnecessary because carbon already includes them. I still have to keep the `::after` selector to hide the `/` though.